src/pal/src/loader/module.cpp

```diff
@@ -676,7 +676,7 @@ PAL_RegisterModule(
 {
     HINSTANCE hinstance = nullptr;
 
-    int err = PAL_InitializeDLL();
+    int err = PAL_InitializeDLLWithoutSEH();
```

Comment:
@mikem8361 Oh, I remember now that the load of mscordac in the soscommand.cpp is just a preload.
But  why do they have the PAL_InitializeDLL in their DllMain then?